### PR TITLE
feat: US131489 Update RIO help pop up wording

### PR DESCRIPTION
**Pull request by martinwang-d2l**
[US131489](https://rally1.rallydev.com/#/?detail=/userstory/608664369071&fdp=true): Update RIO help pop-up wording

Updated UI: Screenshot shown here: 

image

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-retake-incorrect-only-editor.js [hunk 1]
```diff
@@ -68,7 +68,8 @@ class ActivityQuizRetakeIncorrectOnlyEditor
 				?opened="${this.opened}"
 				@d2l-dialog-close="${this.handleClose}"
 				title-text="${this.localize('rioDialogTitle')}">
-					${this.localize('rioDialogParagraph')}
+					<p> ${this.localize('rioDialogParagraph')} </p>
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-retake-incorrect-only-editor.js:71]: There's whitespace characters next to the opening and closing `<p>` tags  that surround the `${this.localize('...')}` calls; can you check they're not introducing unnecessary whitespace in the help pop-up text?

**Review by mostlyfabulous — APPROVED** [review]

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-retake-incorrect-only-editor.js (hunk 1 above)
Comment (martinwang-d2l) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-retake-incorrect-only-editor.js]: Thanks, it didn't affect the output, fixed it anyway.

**Review by mostlyfabulous — APPROVED** [review]
